### PR TITLE
Remove & replace .fancy-link and .fancy-link:hover classes

### LINKED ISSUE #1160: Issue: Transitioning Away from Global Styles in globals.css
## Context

We currently have numerous global style classes defined in the globals.css file, which goes against the component-based styling approach promoted by Tailwind CSS. This can lead to unintended styling consequences throughout the project.

We plan to incrementally remove these styles.
As the first step in moving away from this approach, we need to:

Remove the .fancy-link and .fancy-link:hover classes from globals.css.
Copy their styles and apply them directly within the relevant components where they are used.

## Expected Behavior

Ensure that the updated components are tested before and after the change to verify that the styles (including the hover) remain consistent.

## Additional info
File can be found at:

/styles/globals.css

### PR DESCRIPTION
**Pull request by wolfofdalalst**
# ✨ Codu Pull Request 💻

Fixes #1160 

## Pull Request details
In order to transition away from global  styles, I have removed and `.fancy-link` and `.fancy-link:hover` from `styles/globals.css`, and replaced it with the equivalent tailwindcss classes, as requested in the above mentioned issue. If there are any changes, do let me know, I will make the necessary changes required. I am working under the **hacktoberfest** label.

## Any Breaking changes

None

## Associated Screenshots

None

### CODE REVIEW

File: app/(editor)/create/[[...paramsArr]]/_client.tsx [hunk 1]
```diff
@@ -803,7 +803,7 @@ const Create = ({ session }: { session: Session | null }) => {
                       href="https://www.markdownguide.org/"
                       target="_blank"
                       rel="noopener noreferrer"
-                      className="fancy-link"
+                      className="cursor-pointer bg-gradient-to-r from-orange-400 to-pink-600 bg-clip-text tracking-wide text-transparent"
```
Comment (John-Paul-Larkin) [inline app/(editor)/create/[[...paramsArr]]/_client.tsx]: Is there a reason you didnt add the hover here?

**Comment by wolfofdalalst** [comment]
@John-Paul-Larkin changes have been made.

**Comment by wolfofdalalst** [comment]
@John-Paul-Larkin all checks have passed, please review.

**Review by NiallJoeMaher — APPROVED** [review]
LGTM! 🌮 🦖

**Pull request merged**